lib/fog/google/requests/compute/insert_server.rb

```diff
@@ -151,12 +151,17 @@ def insert_server(server_name, zone_name, options={}, *deprecated_args)
 
           scheduling = {
             'automaticRestart' => false,
-            'onHostMaintenance' => "MIGRATE"
+            'onHostMaintenance' => "MIGRATE",
+            'preemptible' => true
```

Comment:
It seems scary to me that we would default to preemptible VMs, if the user doesn't specify.  That's also a very large change from what the current behavior is.